lib/Runtime/PlatformAgnostic/Platform/Linux/DateTime.cpp

```diff
@@ -188,8 +187,8 @@ namespace DateTime
         // tm doesn't have milliseconds
         int milliseconds = utc->time % 1000;
 
-        tzset();
-        time_t ltime = timegm(&utc_tm);
+        time_t ltime = mktime(&utc_tm);
+        ltime += -(utc_tm.tm_isdst * 3600) + (utc_tm.tm_gmtoff);
```

Comment:
> And the docs seem to indicate that tm_isdst can be negative if it is unknown

We fix the date at the other part of the same file. If system really doesn't know DST for a modern date, we will end-up showing the wrong date.

Besides, same opposite calculation was already being done under LocalToUTC. So, it's a give/take situation.